File: dungeons-26.1.2/src/main/java/fishmod/utils/rendering/DrawEvents.java
package fishmod.utils.rendering;

import fishmod.utils.rendering.drawevents.SlotEvent;

public class DrawEvents {
    public static DrawHandler<SlotEvent> INVENTORY_SLOT_AFTER = new DrawHandler<>();
    public static DrawHandler<SlotEvent> INVENTORY_SLOT_BEFORE = new DrawHandler<>();
    public static DrawHandler<SlotEvent> HUD_SLOT_AFTER = new DrawHandler<>();
    public static DrawHandler<SlotEvent> HUD_SLOT_BEFORE = new DrawHandler<>();
}

File: dungeons-26.1.2/src/main/kotlin/fishmod/utils/rendering/DrawEvents.kt
package fishmod.utils.rendering

import fishmod.utils.rendering.drawevents.SlotEvent

object DrawEvents {
    @JvmField
    var currentSlot: net.minecraft.world.inventory.Slot? = null

    @JvmField
    var INVENTORY_SLOT_AFTER = SimpleHandler<SlotEvent>()
    @JvmField
    var INVENTORY_SLOT_BEFORE = SimpleHandler<SlotEvent>()
}
